packages/mrgn-ui/src/components/action-box-v2/actions/loop-box/hooks/use-loop-simulation.hooks.ts

```diff
@@ -142,7 +146,8 @@ export function useLoopSimulation({
           amount,
           slippageBps,
           marginfiClient.provider.connection,
-          0 //priorityFee
```

Comment:
why no priority fee here? Arent you importing that from the actionboxstore too?

Reply:
priority fee isn't used anymore, we only use bundletip